### DISCUSSION TITLE
Wooden Coaster Stats are Bugged with Inversions

### DISCUSSION THREAD

**Discussion by marscrystalpower** [OpenRCT2/OpenRCT2#23164] [Feature requests]
### Operating System

Windows 10, 64-bit

### OpenRCT2 build

OpenRCT2, v0.4.16 (ff12d31926 on develop) provided by GitHub

### Base game

RollerCoaster Tycoon 2

### Area(s) with this issue?

The bug also exists in the original game (RCT2/RCTC)

### Describe the issue

Almost every coaster in the game allows for the ride length requirement penalty to be ignored when the coaster has at least one inversion. Weirdly, this isn't true for the Wooden Coaster, which leads to bizarre situations where such a coaster (a shuttle loop which launches and does 2 inversions in my example below) has lower stats than... a Twist. Seems unrealistic, if nothing else, especially since Boosters are now present in the game. Unsure if this is a bug, an intended feature by Chris Sawyer, or if it's simply a request to be fixed in future versions of OpenRCT2.

### Steps to reproduce

1. Build a Wooden Coaster that meets every requirement, but fails the 370 meters (1214 feet) length requirement
2. Include an inversion
3. Stats are still shown as Low - Low - Low

### Attachments

![Image](https://github.com/user-attachments/assets/e6da132f-0f42-45e8-b241-a31941d34b5f)
![Image](https://github.com/user-attachments/assets/4ec2b3f1-fcca-4e94-9b4b-07f6536a211d)

**Comment by SpartanFrederic104** [comment]
The Wooden Roller Coaster has a max negative vertical Gs requirement of 0.10 g or lower, and none of its stat requirements can be bypassed with an inversion. This is intentional behavior that was already present in RCT2 vanilla.

**Comment by Gymnasiast** [comment]
Indeed not a bug. I can move this to the Feature Requests section, though, if OP thinks it should be changed.

**Comment by marscrystalpower** [comment]
> Indeed not a bug. I can move this to the Feature Requests section, though, if OP thinks it should be changed.

I see; my mistake. Thanks, yeah, if you could do that. I think it's a bit silly to have the stats show as _that_ low, particularly since the Wooden Coaster has always been capable of inversions in the first place. Perhaps others agree.

**Comment by DelendaEstCarthago** [comment]
Agree with letting inversions nullify the stat penalties for insufficient Ride Length, Max Negative Gs, or Number of Drops (effectively the same as the Corkscrew and Twister coasters). I believe this is also the case with the Vertical Drop coaster, which means neither of the 2 can be used as shuttle loops or micro coasters. "Realism" aside, any coaster with inversions having penalized stats lower than a Gentle ride does seem silly (albeit hilarious)

If others think that the stat penalty is a good thing for these particular rides, maybe it could be less harsh.
